HACKING.md

```diff
@@ -172,4 +172,53 @@ and then a final native binary is produced from this klibrary using the -Xinclud
      teamcity:id:42491947:nativeReport.json
      
  Pay attention, user and password information(with flag `-u <username>:<password>`) should be provided to get data from TeamCity.
-    
+   
+## Composite build and testing
+
+If you have a fix spanning both Kotlin and Kotlin/native workspaces you need to be able to test Native composite build. Here's how to do it manually:
+
+### Have a composite build with the proper Kotlin tag.
+
+Find the version of Kotlin the current native is guaranteed to build with. 
+The version is specified in `kotlin-native/gradle.properties`. For example:
+```
+kotlinVersion=1.3.70-dev-1526
+```
+Checkout `kotlin` workspace to tag `build-1.3.70-dev-1526`.
+Direct `kotlin-native` build to the kotlin with `kotlinProjectPath` in native's `gradle.properties`.
```

Comment:
It's important to note that path to kotlin should mandatory ends with '/kotlin' otherwise build will fail in various unexpected ways.